Soft Pop Adjustments/common/scripted_effects/yMoG_SPA_cache.txt

```diff
@@ -12,6 +12,10 @@ mog_spa_building_cache_update_effect = {
             OR = {
                 has_port = yes
                 has_building = building_trade_center
+                AND = {
+                    Grey_USU_is_active = yes
+                    is_building_type = building_river_port
+                }
```

Comment:
I was under the impression that "has_port" is actually a boolean that checks against the "is_port" criteria on a building, so I am fairly sure this is redundant?

Reply:
That would be "has_building = building_port"